lib/tests/test_utils.cc

```diff
@@ -0,0 +1,56 @@
+#include "test_utils.hpp"
+
+using namespace std;
+
+namespace hocon {
+
+    shared_ptr<simple_config_origin> fake_origin(string description, int line_number) {
+        return make_shared<simple_config_origin>(move(description), line_number, line_number, origin_type::GENERIC);
+    }
+
+    shared_ptr<value> string_token(string const& text, config_string_type type) {
```

Comment:
This is definitely not significant, but normally it's better to take string by value in a constructor if you plan to take ownership of it (as config_string does, which you'd need to do move(text) to). You do that most places, so this looks like an oversight.